pytorch_lightning/metrics/convertors.py

```diff
@@ -0,0 +1,207 @@
+import numbers
```

Comment:
Do you mind changing the spelling to converter? I originally thought this was just misspelled but then realized you can spell converter this way. However, I do think converter is better considering that's what most people call it for this use case (i.e. search "Converter design pattern".

Reply:
Done